### ISSUE TITLE
Model default scale

### ISSUE THREAD

**Issue by michbil** [HippoAR/react-native-arkit#147]
I've tried to add model to my scene, and was struggling for some time, because model was invisible:

```
 <ARKit.Model
            position={{ x: -0.2, y: 0, z: 0, frame: 'local' }}
            scale={0.01}
            model={{
              file: 'art.scnassets/ship.scn', // make sure you have the model file in the ios project
            }}
          />
```

After some digging i've found if scale is not set this way, model is invisible: 
```
 <ARKit.Model
            position={{ x: -0.2, y: 0, z: 0, frame: 'local' }}
            model={{
              file: 'art.scnassets/ship.scn', // make sure you have the model file in the ios project
              scale: 0.01,
            }}
          />
```

I think we need update project README.md, so it will be more clear how to use model component.

**Comment by macrozone** [comment]
hi @michbil thx for pointing this out!

actually the scale property on the model is deprecated, that's why i removed it from the README. Now every node (Box, Torus, Model, etc.) has a scale property.

The bug is actually that the scale on the model defaults incorrectly to 0 instead of 1. I try to fix that

**Comment by macrozone** [comment]
I released 0.8.0-beta.4 that no longer needs the scale property on model
